packages/swap/src/routes/v2/utils.ts

```diff
@@ -83,7 +83,10 @@ export const getDepositInfo = (
       amount,
       txRef: depositTransactionRef,
       txConfirmations: pendingDeposit?.transactionConfirmations,
-      witnessedAt: swapRequest?.depositReceivedAt?.valueOf(),
+      witnessedAt:
+        swapRequest?.originType === 'VAULT'
+          ? swapRequest.swapRequestedAt.valueOf()
+          : swapRequest?.depositReceivedAt?.valueOf(),
```

Comment:
We don't have a `depositReceivedAt` entry for smart contract swaps therefore defaulting to the `swapRequestedAt`